source/reviews/comments.txt

```diff
@@ -0,0 +1,112 @@
+==========================================
+How To Write Code And Copy Review Comments
+==========================================
+
+Summary
+-------
+
+- Be kind.
+- Explain your reasoning.
+- Balance giving explicit directions with just pointing out problems and letting
+  the author decide.
+- Encourage authors to simplify code or add code comments instead
+  of just explaining the complexity to you.
+- Encourage authors to simplify explanations when appropriate. We're all
+  smart, but smart words take more time to read. Readers prefer
+  conciseness and the simplest explanation in most situations.
```

Comment:
Comment: I think this guideline conflates "concise" with "simple". I believe this ideas should be discussed separately ([short snake code to illustrate point](https://github.com/guckstift/shortest-js-snake/blob/master/index.html))

Reply:
I need to simplify this